Makefile

```diff
@@ -110,10 +110,6 @@ else
 	bash sanity/check_sanity.sh
 endif
 
-crio:
-	bash .ci/install_bats.sh
-	RUNTIME=${RUNTIME} ./integration/cri-o/cri-o.sh
```

Comment:
The CI job http://jenkins.katacontainers.io/job/kata-containers-crio-PR calls that `integration/cri-o/crio-o.sh` to run the cri-io functional tests, however currently is configured for Kata 1.x. While this change has no impact on that job, I am wondering what is the strategy for when we enable a similar job for 2.x. More precisely, even for a 2.x job we won't call that script but instead run a k8s+crio (running the k8s conformance) job. Did I understand it correctly?

Reply:
oh, also I think we won't need the `TEST_CRIO` variable anymore. So need to adjust `.ci/jenkins_job_build.sh`. Such as change I would defer to after today's release.

Reply:
@wainersm, you got that correctly.  The whole integration between CRI-O and Kata Containers will be done differently for 2.x.
The part on the kata-containers repo that we'll use, is based in the `integration/kubernetes`, and the part that uses `integration/cri-o/cri-o.sh` will be implemented entirely in the `prow`, in the OpenShift CI.

Does this make sense? :-)